src/components/flowchart/styles/_variables.scss

```diff
@@ -26,7 +26,7 @@
   @mixin export {
     --node-fill-default: #{colors.$white-200};
     --node-stroke-default: #{colors.$white-900};
-    --edge-stroke: colors.$white-900;
+    --edge-stroke: #{colors.$black-100};
     --edge-arrowhead-fill: colors.$white-900;
```

Comment:
This might also need to be wrapped in `#{ }` as well.

Reply:
I removed the arrow head style as it is not needed